src/app/auth/auth.component.ts

```diff
@@ -71,7 +69,6 @@ export class AuthComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.logger.startSession();
-    this.isReady = false;
```

Comment:
Even though `isAppOutdated` and `versionCheckingError` and `isSettingUpSession` is implicitly initialized to false, I think it is good to do it explicitly following the convention for `isReady`.

Reply:
Agreed, it would be better to state explicitly! 😄

Reply:
Made the changes 💯

Reply:
will fix the linting tonight 😅